tests/test_ir.py

```diff
@@ -33,14 +49,22 @@ def v_num(self):
         return v2, v3, v4, v11, v13, v23
 
     @pytest.fixture
-    def v_literal(self):
+    def v_literal(self, x, y):
         vx = Vector(x)
         vxy = Vector(x, y)
         vx1y = Vector(x + 1, y)
         s = Scalar(name='s', nonnegative=True)
         vs3 = Vector(s + 3, smart=True)
         return vx, vxy, vx1y, vs3
 
+    @pytest.fixture
+    def fa(self, grid, x):
+        return Array(name='fa', dimensions=(x,), shape=(3,)).indexed
+
+    @pytest.fixture
+    def fc(self, grid, x, y):
+        return Array(name='fc', shape=(3, 5), dimensions=(x, y)).indexed
```

Comment:
Fixing argument order for consistency

Reply:
It's kwargs doesn't really matter, I'll try to remember to change it though.